src/ui/server/controllers/insurance/account/sign-out/index.ts

```diff
@@ -0,0 +1,22 @@
+import { INSURANCE_ROUTES } from '../../../../constants/routes/insurance';
+import { Request, Response } from '../../../../../types';
+
+const {
+  ACCOUNT: {
+    SIGN_IN: { ROOT: SIGN_IN_ROOT },
+  },
+} = INSURANCE_ROUTES;
+
+/**
+ * get
+ * Account sign out / wipe user session
+ * @param {Express.Request} Express request
+ * @param {Express.Response} Express response
+ * @returns {Express.Response.redirect} Redirect to sign in page
+ */
+export const get = (req: Request, res: Response) => {
+  delete req.session.user;
```

Comment:
Do we need to check whether `session.use` exists before we delete it?

Reply:
No, it will work even if it's undefined. If it was more than 1 level we would have to check